beryllia/__init__.py

```diff
@@ -109,6 +111,43 @@ async def _is_oper(self, nickname: str):
         })
         return whois_line.command == RPL_WHOISOPERATOR
 
+    async def _find_unset(self):
+        # check stats k (locally set kline) and g (kline set on another server)
+        # to see if anything was unset while we were gone
+        known_klines  = await self.database.list_klines()
+        active_klines = set()
+        await self.send(build("STATS", ["k"]))
+
+        stats_kline = Response(RPL_STATSKLINE, [SELF, ANY])
+        stats_end   = Response(RPL_ENDOFSTATS, [SELF, ANY])
+        #:lithium.libera.chat 216 sandcat sandcat :k 1.2.3.4 * * :piss off m8 (2021/8/23 20.24)|(launchd!launchd@localhost{launchd})
+        #:lithium.libera.chat 219 sandcat sandcat :k :End of /STATS report
```

Comment:
these examples appear to be incorrect. from solanum code it looks like it should be 
```python
#:lithium.libera.chat 216 sandcat k 1.2.3.4 * * :piss off m8 (2021/8/23 20.24)|(launchd!launchd@localhost{launchd})
#:lithium.libera.chat 219 sandcat k :End of /STATS report